build_guest

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -fvx 
+
+wget https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.12.5.tar.xz
+mkdir -p linux
+cd linux && xzcat ../linux-6.12.5.tar.xz | tar xf -
+cd linux-6.12.5
+cp ../../cpu.config .config
```

Comment:
If we want to support using an existing user source directory then not sure I'd do it this way, but instead specify a config name and then parameterize the config during boot.